examples/info.rb

```diff
@@ -16,6 +16,13 @@
   puts "  Block align:           #{reader.native_format.block_align}"
   puts "  Sample frame count:    #{reader.total_sample_frames}"
 
+  unless reader.sample_info.nil?
```

Comment:
It seems like it would be useful to include the other fields from the `smpl` chunk here as well. (Thanks for adding this to `info.rb`!)